virt/logging_events_monitoring/virt-reviewing-virtualization-overview.adoc

```diff
@@ -0,0 +1,30 @@
+[id="virt-reviewing-virtualization-overview"]
+= Reviewing Virtualization Overview
+include::modules/virt-document-attributes.adoc[]
+:context: virt-reviewing-virtualization-overview
+
+toc::[]
+
+The *Virtualization Overview* page provides a comprehensive view of virtualization resources, details, status, and top consumers. By gaining an insight into the overall health of {VirtProductName}, you can determine if intervention is required to resolve specific issues identified by examining the data.
+
+Use the *Getting Started* resources to access quick starts, read the latest blogs on virtualization, and learn how to use operators. Obtain complete information about alerts, events, inventory, and status of virtual machines. Customize the *Top Consumer* cards to obtain data on high utilization of a specific resource by projects, virtual machines, or nodes. Click *View virtualization dashboard* for quick access to the xref:../../virt/logging_events_monitoring/virt-reviewing-vm-dashboard.adoc#virt-reviewing-vm-dashboard[Dashboards] page.
+
+[id="prerequisites_{context}"]
+== Prerequisites
+
+To use the *vCPU wait* metric in the *Top Consumers* card, the `schedstats=enable` kernel argument must be applied to the `MachineConfig` object. This kernel argument enables scheduler statistics used for debugging and performance tuning and adds a minor additional load to the scheduler. See the xref:../../post_installation_configuration/machine-configuration-tasks.adoc#nodes-nodes-kernel-arguments_post-install-machine-configuration-tasks[{product-title} machine configuration tasks] documentation for more information on applying a kernel argument.
```

Comment:
If this prerequisite applies only to "vCPU wait", maybe it could be added as a note in virt-resources-monitored-for-top-consumption.adoc. Or it could be kept in the assembly and added as a note after that module if you want to retain the machine configuration link.